app_data/assets/contents.json

```diff
@@ -15,6 +15,10 @@
     "md5Checksum": "5c2cf4dfa421bce1c99eabf15ad660b8",
     "size_kb": 290.4
   },
+  "contents.json": {
+    "md5Checksum": "76bbc25182af1bedbf8505a283877e25",
+    "size_kb": 2.7
+  },
```

Comment:
@jfmcquade After I changed the config so that `plh_facilitator_mx` is now an extended deployment of `plh_facilitator`, the entry `contents.json` shows up in `contents.json`. 

Something strange happening with the inheritance / nesting? I don't think the fact that this appears actually breaks anyting.

Reply:
That is strange, I can confirm that I can recreate: this entry shows up when I run `yarn workflow sync_assets` when the config is the child of `plh_facilitator`, but not when it is independent. 

It shouldn't cause any issues, as this contents file is just used for looking up the assets, but would be nice to understand why it's happening. I can have a little look tomorrow.